Versionize/Lifecycle/BumpFileUpdater.cs

```diff
@@ -0,0 +1,46 @@
+﻿using NuGet.Versioning;
+using Versionize.Config;
+using static Versionize.CommandLine.CommandLineUI;
+
+namespace Versionize;
+
+public sealed class BumpFileUpdater
+{
+    public static void Update(
+        Options options,
+        SemanticVersion nextVersion,
+        IBumpFile bumpFile)
+    {
+        if (options.SkipCommit)
```

Comment:
In v1, `--skip-commit` increment the version number of project files.
Wouldn't we keep this behavior ?

In my opinion, when I read the `--skip-commit` parameter, I understand that it skips the commit only (and still increments the version in the project files).

Reply:
My logic for doing this at the time was "what's the point of modifying a release-related file if you're not going to commit it in the release commit"?

But I guess you're interested in the use case of creating your own release commit instead of versionize doing it for you? So you can customize the message or include additional files in the commit?

Reply:
> But I guess you're interested in the use case of creating your own release commit instead of versionize doing it for you? So you can customize the message or include additional files in the commit?

Yes, that's right. There's a need to be able to check and modify before publishing the version.

Reply:
Yeah, you're right. I've done the same on certain projects for including stuff like benchmark results in release commits (but with tools other than Versionize). Thanks for pointing this out.